.rubocop.yml

```diff
@@ -191,16 +207,13 @@ Metrics/ClassLength:
 
 
 # Configuration parameters: AllowURI, URISchemes.
-Metrics/LineLength:
+Layout/LineLength:
   Max: 370
 
 # Configuration parameters: CountKeywordArgs.
 Metrics/ParameterLists:
   Max: 17
 
-Metrics/PerceivedComplexity:
-  Max: 18
```

Comment:
It seems that logic was changed and some files don't meet this threshold any more. So I removed it here and rubocop put "Max: 26" in the TODO file.